config/loader_hcl2.go

```diff
@@ -0,0 +1,461 @@
+package config
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	gohcl2 "github.com/hashicorp/hcl2/gohcl"
+	hcl2 "github.com/hashicorp/hcl2/hcl"
+	hcl2parse "github.com/hashicorp/hcl2/hclparse"
+	"github.com/zclconf/go-cty/cty"
```

Comment:
Is cty tied to hcl at all? (I can't remember if that was going to be added to the hcl repo or not)

Reply:
For now I'm planning to leave it where it is, because there's no strong need for it to be in the same repo and there's a lot in there to pick apart and little advantage in doing so. We can decide in the next phase if we want to do some more refactoring in this area.